src/pyodide.js

```diff
@@ -523,64 +566,6 @@ globalThis.loadPyodide = async function(config = {}) {
     return Module.pyodide_py.eval_code(code, globals);
   };
 
-  // clang-format off
```

Comment:
What was the motivation for moving this code block around? It would make using git blame more difficult in the future.

Reply:
> but I think it's helpful for them to be next to each other in the source too.

I'm OK with the change here if you find it useful. In future PRs though it is really helpful to go back in git history with git blame to understand changes, and moving functions around kind of breaks that workflow. It's still possible, just that the resulting improvement need to be worth that cost.

> It's also possible to sort API docs based on the order in PUBLIC_API

Maybe we could sort them alphabetically as autodoc would do by default https://www.sphinx-doc.org/en/master/usage/extensions/autodoc.html#confval-autodoc_member_order ?

Reply:
In this case I think the small benefit is almost exactly matched by the small cost... \*shrug\*

So there are three options for sort: sort as they appear in `pyodide.js`, sort alphabetically, or sort by `PUBLIC_API`. The reason I didn't sort alphabetically is that sometimes it sorts closely related methods far away from each other (e.g., `register` / `unregister`). It's a pretty minor issue, I'm happy to just pick whatever and stick with it.

Reply:
Okay I reverted this, will sort js API docs alphabetically.